pyuia/appium/context.py

```diff
@@ -1,9 +1,14 @@
-from ..selenium import SeleniumContext
+from pyuia import AppContext
 from .util import get_logs
 
 __all__ = ['AppiumContext']
 
-class AppiumContext(SeleniumContext):
```

Comment:
I think the relationship between `AppiumContext` and `SeleniumContext` cannot be broken. How about implementing `platform` as a property, in which the platform is extracted from capabilities dynamically and lazily?